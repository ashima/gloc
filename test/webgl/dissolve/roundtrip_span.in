// Copyright 2012 Foo Corp <> All Rights Reversed.
// Version: 0.0.1
// Author: A
// Author: B
// Author: C
// Build: r4929f <http://www.example.com/>

void f(void);

//End: Copyright

void j(void);

// Copyright 2012 Bar Corp <> Something
// Version: 0.1.0 <mailto:bar@example.org>
// License: BSD-3-Clause
// Library: Magic Box <http://example.com/magic>
#line 1 1
void main() {
  gl_Position = vec4(1.);
}
