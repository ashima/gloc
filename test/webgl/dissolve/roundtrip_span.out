[
  [
    "[stdin]",
    {
      "glo": [ 1, 0, 0 ],
      "target": [ "webgl", [ 1, 0, 0 ] ],
      "meta": {
        "copyright": [ 2012, [ "Foo Corp", "" ] ],
        "author": [ [ "A", "" ], [ "B", "" ], [ "C", "" ] ],
        "version": [ [ 0, 0, 1 ], "" ],
        "build": [ "r4929f", "http://www.example.com/" ]
      },
      "units": [
        {
          "insym": [ "f" ],
          "source": "#line 8 GLOC_0\n       void f ( void ) ;"
        },
        {
          "insym": [ "j" ],
          "source": "#line 12 GLOC_0\n       void j ( void ) ;"
        }
      ],
      "linkmap": { "0": "#n=0" }
    }
  ],
  [
    "[stdin]",
    {
      "glo": [ 1, 0, 0 ],
      "target": [ "webgl", [ 1, 0, 0 ] ],
      "meta": {
        "copyright": [ 2012, [ "Bar Corp", "" ] ],
        "license": [ "BSD-3-Clause", "" ],
        "library": [ "Magic Box", "http://example.com/magic" ],
        "version": [ [ 0, 1, 0 ], "mailto:bar@example.org" ]
      },
      "units": [
        {
          "outsym": [ "main" ],
          "source":
            "#line 1 GLOC_1\nvoid main() {\n  gl_Position = vec4(1.);\n}"
        }
      ],
      "linkmap": { "1": "#n=1" }
    }
  ]
]
